creator/nx_creator_xpcs.py

```diff
@@ -14,20 +15,20 @@
 NX_APP_DEF_NAME = "NXxpcs"  # name of NeXus Application Definition
 
 
-# TODO: not defined in NeXus at this time
+#TODO: not defined in NeXus at this time
 # Need to create & propose this new Application Definition to NIAC
 
 
 class NXCreator:
     """
-
-    Write a NeXus file from the XPCS data
+    Write a NeXus file from the XPCS and SAXS data
```

Comment:
all one line, spell correction for `definition`

Reply:
such as: `Write NeXus file from XPCS & SAXS data using NXxpcs & NXcansas definitions`